packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -37,6 +38,9 @@ export default class Extension {
 
   readonly #state: State;
 
+  private cachedAccount: AutoSavedAccount | undefined;
+  private cacheTimeOut: number | undefined;
```

Comment:
```suggestion
  #cachedAccount: AutoSavedAccount | undefined;
  #cacheTimeOut: number | undefined;
```

Rather use Js privates since TS supports it.